pkg/cmd/template.go

```diff
@@ -108,15 +108,21 @@ func newTemplateCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 			// we always want to print the YAML, even if it is not valid. The error is still returned afterwards.
 			if rel != nil {
 				var manifests bytes.Buffer
-				fmt.Fprintln(&manifests, strings.TrimSpace(rel.Manifest))
+				_, err := fmt.Fprintln(&manifests, strings.TrimSpace(rel.Manifest))
+				if err != nil {
+					return err
+				}
```

Comment:
`bytes.Buffer` never returns an error. The `if err != nil` will never be run because the underlying writer to the buffer doesn't produce an error. It always returns nil to follow the interface.

This was something I learned about Go when I went digging into the source.

Can we remove the error handling around `bytes.Buffer` cases since it will never execute?